### ISSUE TITLE
Mismatch of colvars outputs when "wrapAll" is turned on

### ISSUE THREAD

**Issue by tabrischunchan** [colvars/colvars#40]
As I used to turn "wrapAll on" in the namd configuration file and I expected it does no effect on the calculation. However when it combines with colvars, which affects the calculation, I am confused what will colvars read in after continuing from a previous dcd (wrapped). 

In fact, I have done continuing simulation, namely run-1 then run-2, twice with "wrapAll on" and with "wrapAll off". It turns out the one with "wrapAll off" will give reasonable outputs in which the last line of colvars.traj of run-1 is as same as the first line of colvars.traj of run-2. On the other hand, the one with "wrapAll on" does not behave like this. 

I will attach my colvar.in and configuration file but they are just conventional things. Still, hope they will help.

colvars.in:
```
colvarsTrajFrequency     1000
colvarsRestartFrequency  1000

colvar {
   name dih

   width 1.0
   outputValue on

   dihedral {
       group1 {
         atomsFile group2.pdb
         atomsCol B
         atomsColValue 1.0
       }
       group2 {
         atomsFile group1.pdb
         atomsCol B
         atomsColValue 1.0
       }
       group3 {
         atomsFile group3.pdb
         atomsCol B
         atomsColValue 1.0
       }
       group4 {
         atomsFile group4.pdb
         atomsCol B
         atomsColValue 1.0
       }
       oneSiteSystemForce yes
   }

   outputSystemForce yes
   outputAppliedForce yes
}

harmonic {
  name diah_pot
  colvars dih
  centers -11.0
  forceConstant 10.0
}
```

run.namd:
```
firsttimestep 0

structure       ../ionized.psf
coordinates     ../ionized.pdb

Bincoordinates  ./output/NPT-0.restart.coor
Binvelocities   ./output/NPT-0.restart.vel
ExtendedSystem  ./output/NPT-0.restart.xsc

outputname       ./output/NPT-1

COMmotion yes

paraTypeCharmm      on
parameters          ../par_CMAP.inp

exclude             scaled1-4
1-4scaling          1.0
cutoff              12.0
switching           on
switchdist          10.0
pairlistdist        13.5

timestep            2.0
rigidBonds          all
nonbondedFreq       1
fullElectFrequency  2
stepspercycle       20

langevin            on
langevinDamping     5.0
langevinTemp        310.0
langevinHydrogen    off

wrapAll             on

PME                 yes
PMEGridSpacing 1.0

langevinPiston        on
langevinPistonTarget  1.01325
langevinPistonPeriod  2000.0
langevinPistonDecay   2000.0
langevinPistonTemp    310.0

restartfreq         1000
dcdfreq             1000
xstFreq             1000
outputEnergies      100
outputPressure      100

colvars             on
```

**Comment by giacomofiorin** [comment]
Unless all atoms in a group belong to the same molecule, that group's
center of mass will not be calculated correctly when using wrapAll.  Please
disable wrapAll and resume the simulation from unwrapped coordinates.

**Comment by tabrischunchan** [comment]
Unfortunately all my 4 groups of atoms belong to the same molecule. The simulation was simply a single molecule relaxing in explicit water. I have also examined both wrapped and unwrapped trajectories, in fact the molecule was staying in the center of the box all the time so PBC issue should not exist. I did solve the problem with disabling wrapAll but I just do not see any reason why it went wrong with wrapAll turned on.
